nextcord/role.py

```diff
@@ -261,6 +266,8 @@ def _update(self, data: RolePayload) -> None:
         except KeyError:
             self.tags = None
 
+        self.flags: Optional[RoleFlags] = data.get("flags")
```

Comment:
Are you sure this would work? `data.get("flags")` is an integer, `RoleFlags` is a `Flag`.